.drone.yml

```diff
@@ -6,7 +6,7 @@ matrix:
   MINOR:
     - 0
   PATCH:
-    - 5
```

Comment:
Does this count as a feature? If so, it would be a minor bump right?

Reply:
Changed...